test/e2e/atlas/kubernetes_config_apply_test.go

```diff
@@ -67,10 +69,22 @@ func TestKubernetesConfigApply(t *testing.T) {
 			operator.restoreOperator()
 		})
 
+		e2eNamespace := &corev1.Namespace{
+			ObjectMeta: apisv1.ObjectMeta{
+				Name: "e2e-autodetect-parameters",
+			},
+		}
+		t.Logf("adding namespace %s", e2eNamespace)
+		require.NoError(t, operator.createK8sObject(e2eNamespace))
+		g.t.Cleanup(func() {
+			req.NoError(operator.deleteK8sObject(e2eNamespace))
+		})
+
 		cmd := exec.Command(cliPath,
 			"kubernetes",
 			"config",
 			"apply",
+			"--targetNamespace", "e2e-autodetect-parameters",
```

Comment:
is this flag now mandatory or was it added here for the test

Reply:
it was added as concurrently executing tasks stomped on each other in the same namespace.